ibm_db2/datadog_checks/ibm_db2/ibm_db2.py

```diff
@@ -571,13 +571,15 @@ def get_connection(self):
             return connection
 
     @classmethod
-    def get_connection_data(cls, db, username, password, host, port, tls_cert):
+    def get_connection_data(cls, db, username, password, host, port, security, tls_cert):
```

Comment:
The `security` parameter needs to be passed to the method call on line [556](https://github.com/DataDog/integrations-core/pull/11236/files#diff-8d24fdc843cf34d169cc51a1e0d7d1384dda47cb12eee1322f869118cb03928aR556). Can you also define `self._security` in the init method [here](https://github.com/DataDog/integrations-core/pull/11236/files#diff-8d24fdc843cf34d169cc51a1e0d7d1384dda47cb12eee1322f869118cb03928aR33)?